distribution/lib/Standard/Database/0.0.0-dev/src/DB_Table.enso

```diff
@@ -484,6 +484,8 @@ type DB_Table
        - column_map: Mapping from old column names to new or a vector of new
          column names to apply by position. `Regex` objects can be used
          within the mapping to do pattern based renaming.
+         Can also be supplied as a `Table` either with a single column of new
+         names or two columns with old and new names.
```

Comment:
I'm wondering if we should mention that 1st column is used for Old names and 2nd for new names and column names are ignored.

As I'd be wondering what the user may expect if they give a table with columns `['New Name', 'Key']` for example.